src/setup.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-console */
+import 'dotenv/config';
+import chalk from 'chalk';
+import { sequelize } from './utils/db.js';
+import './models/user.model.js';
+import './models/token.model.js';
+
+sequelize.sync({ force: true }).finally(() => {
```

Comment:
Using `force: true` in `sequelize.sync()` will drop and recreate all tables every time the application starts. This is usually not recommended for production environments as it can lead to data loss. Consider using it only in development or testing environments.